integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java

```diff
@@ -364,4 +371,18 @@ private DomainResource createDomainResource(String domainUid, String domNamespac
     setPodAntiAffinity(domain);
     return domain;
   }
+  
+  private static void enableStrictMode(String namespace) {
+    assertDoesNotThrow(() -> {
+      copyFile(Paths.get(RESOURCE_DIR, "istio", "istio-tls-mode.yaml").toFile(),
+          Paths.get(WORK_DIR, "istio-tls-mode.yaml").toFile());
+    });
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString(),
+          "NAMESPACE", namespace);
+    });
+    ExecResult result = assertDoesNotThrow(() -> ExecCommand.exec(KUBERNETES_CLI + " apply -f "
+        + Paths.get(WORK_DIR, "istio", "istio-tls-mode.yaml").toString(), true));
```

Comment:
remove "istio" parameter here. should be 
Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString()

Reply:
Fixed.